examples/internet-gateway/main.tf

```diff
@@ -30,10 +29,11 @@ module "gateway" {
   # Optional inputs #
   ###################
 
+  # Whether to attach the instances to a public subnet.
+  # attach_public_subnet = true
```

Comment:
@bmanifold This example module doesn't override this setting, so a public subnet will be attached in the main module.